src/domain/Buttons/FontAwesomeIconButton/FontAwesomeIconButton.tsx

```diff
@@ -52,10 +62,14 @@ const FontAwesomeIconButton = (props: IFontAwesomeIconButtonProps) => {
     isHovered = false,
     isDisabled = false,
     componentContext,
+    iconSize = IconSize.Small,
     size = Size.Small,
-    buttonOverrides
+    buttonOverrides,
+    statusDotVariant
   } = props
 
+  const hasStatusDot = !!statusDotVariant && iconSize === IconSize.Large
```

Comment:
It seems that StatusDot is only visible when iconSize is large, but this is kind of hidden information,
How about you show statusDot when statusDotVariant exists, but make it smaller depending on iconSize?

Reply:
It should not have a status dot when the icon size is small as it is unable to see the status dot or the icon behind it. The component library would not add in future cases or discourage pattern to make design consistency.

Reply:
Can you add something to the example file to explain that this is only for large `iconSize` ? It's not clear when reading the docs that this is the case.
This is potentially creating a frustrating developer experience at the expense of design consistency... which I guess its fine but at the same time I really don't care what size buttons people are using status dots on. It seems kind of arbitrary to restrict it to just the large ones.